portfolio/src/main/webapp/about.css

```diff
@@ -0,0 +1,133 @@
+html, body {
```

Comment:
Please alphabetize declaration order (for the whole file): https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
REalphabetize